content/en/docs/instrumentation/net/manual.md

```diff
@@ -210,6 +211,53 @@ In the preceding example, `childOperation` is ended because the scope of the
 `using` block is explicitly defined, rather than scoped to `DoWork` itself like
 `parentOperation`.
 
+## Creating independent Activities
+
+The previous examples showed how to create Activities that follow a nested
+heirarchy. In some cases, you'll want to create inactive Activities that are
+siblings of the same root rather than being nested.
+
+```csharp
+public static void DoWork()
+{
+    using var currentActivity = MyActivitySource.StartActivity("MyActivity");
+
+    using var sibling1 = MyActivitySource.StartActivity(
+        "Sibling1",
+        ActivityKind.Internal, 
+        parentId: currentActivity?.Id);
```

Comment:
Should this be using parentId or parentContext? Both are functionally equivalent in my sample app.

Reply:
please using context.

Reply:
Done. One thing that sorta sucks is because the system.diagnostics api returns nullable values, this requires handling null. Currently I'm doing it inline rather than check up top or use the unsafe `!` operator.